contribs/gmf/src/directives/mobilegeolocation.js

```diff
@@ -0,0 +1,187 @@
+goog.provide('gmf.MobileGeolocationController');
+goog.provide('gmf.mobileGeolocationDirective');
+
+goog.require('gmf');
+goog.require('ngeo.DecorateGeolocation');
+goog.require('ngeo.FeatureOverlay');
+goog.require('ngeo.FeatureOverlayMgr');
+goog.require('ol.Feature');
+goog.require('ol.Geolocation');
+goog.require('ol.Map');
+goog.require('ol.geom.Point');
+goog.require('ol.proj');
+
+
+/**
+ * Provide a "mobile geolocation" directive.
+ *
+ * @example
+ * <gmf-mobile-geolocation
+ *     gmf-mobile-geolocation-map="ctrl.map"
+ *     gmf-mobile-geolocation-options="ctrl.mobileGeolocationOptions">
+ * </gmf-mobile-geolocation>
+ *
+ * @return {angular.Directive} The Directive Definition Object.
+ * @ngInject
+ */
+gmf.mobileGeolocationDirective = function() {
+  return {
+    restrict: 'E',
+    scope: {
+      'getMobileMapFn': '&gmfMobileGeolocationMap',
+      'getMobileGeolocationOptionsFn': '&gmfMobileGeolocationOptions'
+    },
+    controller: 'GmfMobileGeolocationController',
+    controllerAs: 'ctrl',
+    templateUrl: 'partials/mobilegeolocation.html'
```

Comment:
I saw the discussions regarding the caching (pre-loading) of the template files.  Is this comment still up-to-date ?  It would seem to me that it's okay to use files, especially when the string would make the line longer than 80, as they are more versatile to contain large content.

Please, let me know if I must change this to use text instead of a file.